test/FlatLaunchpeg.test.ts

```diff
@@ -122,6 +122,12 @@ describe('FlatLaunchpeg', () => {
       expect(await flatLaunchpeg.balanceOf(bob.address)).to.eq(1)
     })
 
+    it('Mint reverts when not started yet', async () => {
+      await initializePhasesFlatLaunchpeg(flatLaunchpeg, config, Phase.NotStarted)
+
+      await expect(flatLaunchpeg.connect(bob).allowlistMint(1)).to.be.revertedWith('Launchpeg__WrongPhase()')
```

Comment:
can you make sure publicSaleMint also revert in this test please?

Reply:
Done